### PR TITLE
ManageWikiSettings, LocalSettings: Add wgDisplayTitleHideSubtitle

### PR DESCRIPTION
**Pull request by MusikAnimal**
Enable for solarawiki

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -501,6 +501,12 @@
 		'default' => true,
 	],
 
+	// Display Title
+	'wgDisplayTitleHideSubtitle' => [
+		'default' => false,
+		'solarawiki' => true,
```
Comment (paladox) [inline LocalSettings.php:507]: You can remove this, then do it in Special:ManageWiki/settings when merged

File: ManageWikiSettings.php [hunk 2]
```diff
@@ -462,6 +462,15 @@
 		'section' => 'edit',
 		'help' => 'Restrict {{DISPLAYTITLE}} to titles that normalize to the same canonical database key. Wikis with NoTitle extension installed have this config unset.',
 	],
+	'wgDisplayTitleHideSubtitle' => [
+		'name' => 'Don\'t display the page\'s original title below the display title',
+		'restricted' => false,
+		'from' => 'displaytitle',
+		'type' => 'check',
+		'overridedefault' => true,
```
Comment (paladox) [inline ManageWikiSettings.php:470]: Should be false to match the default in ls

File: LocalSettings.php (hunk 1 above)
Comment (MusikAnimal) [reply LocalSettings.php]: Bah, of course!

File: ManageWikiSettings.php (hunk 2 above)
Comment (MusikAnimal) [reply ManageWikiSettings.php]: Done

**Pull request merged**